doc/en/pools.rst

```diff
@@ -374,3 +374,54 @@ The code above specifies a collections of parameters in `@task_target`, and each
 entry will be used create one task - thus 3 tasks will be created from the target.
 
 For a complete and downloadable example, see :ref:`here <example_discover>`.
+
+
+.. _auto_part:
+
+Auto-Part
```

Comment:
Let's move this section to Task discover.

We need a better name for this feature as a whole, let's call it "Auto-Part and Smart Scheduling". 

This feature is enabled by providing the --runtime-data cmdline arg, and then plan does 3 things:
1) automatically determinate the optimal number of parts for multitests from historical runtime data, for part="auto" task target
2) set weight for each task with its historical runtime
3) automatically determine pool size